Core/src/org/sleuthkit/autopsy/timeline/ui/detailview/DetailViewPane.java

```diff
@@ -433,9 +446,15 @@ protected void cancelled() {
         }
 
         @Override
-        protected void setDateAxisValues(Interval timeRange) {
+        protected void setDateValues(Interval timeRange) {
             detailsChartDateAxis.setRange(timeRange, true);
             pinnedDateAxis.setRange(timeRange, true);
         }
+
```

Comment:
no, actually it can't unless I make a ChartRefreshTask to parallel AbstractTimelineChart...